HW_1/Task_2.py

```diff
@@ -0,0 +1,17 @@
+# Задача 4: Петя, Катя и Сережа делают из бумаги журавликов. Вместе
+# они сделали S журавликов. Сколько журавликов сделал каждый
+# ребенок, если известно, что Петя и Сережа сделали одинаковое
+# количество журавликов, а Катя сделала в два раза больше журавликов,
+# чем Петя и Сережа вместе?
+
+# 6 -> 1 4 1
+# 24 -> 4 16 4
+# 60 -> 10 40 10
+
+print("Введите, сколько журавликов дети сделали вместе: ")
+S = int(input())
+if S % 6 == 0:
+    print("Петя и Сережа сделали по", round(S / 6), "журавликов, ")
+    print("а Катя сделала", round(S / 6 * 4), "журавликов")
```

Comment:
Наверное проще будет так
```suggestion
    print("Петя и Сережа сделали по", S // 6, "журавликов, ")
    print("а Катя сделала", S // 6 * 4, "журавликов")
```